### DISCUSSION TITLE
ECE%40Duke/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#253] [Announcements]
# ECE%40Duke/

None

https://opencs.app/ECE%40Duke/

**Comment by Anthea-HJ** [comment]
补充：亲测MS可以用Interfolio，MEng不行（发邮件问过了对方明确表示用不了555）

**Comment by Fridge003** [comment]
24Fall开始会有面试

**Comment by Skylar-xty** [comment]
请问转博情况怎么样？

**Reply by WalterJ726** [reply]
得和老师network，大多数能读PhD去的都是低于或者等于Duke的学校

**Comment by WalterJ726** [comment]
MEng有Coop政策了

**Comment by StelenaChen** [comment]
一般进面试，得到offer几率很高，面试主要考察口语日常交流，不太问专业技术问题，题库不大，小红书上都能搜到，同时面试官也会根据你自己的个人简历，问一些个人经历相关的问题

**Comment by miaaa-z** [comment]
请问找工情况怎么样？我看有很多人说找工的上岸率非常低

**Comment by WalterJ726** [comment]
主要是来这个项目的人都是转码比较多。找工率在前几年还不错，现在转码时代过去之后比较低了。找工主要看自己，需要在一开始的时候就刷好题准备好面试

Mia ***@***.***> 于2026年4月2日周四 02:38写道：
